bigquery_etl/parse_metadata.py

```diff
@@ -94,3 +94,7 @@ def is_public_json(self):
     def is_incremental(self):
         """Return true if the incremental flag is set."""
         return "incremental" in self.labels
+
+    def is_incremental_export(self):
+        """Return true if the incremental_export flag is set."""
+        return "incremental_export" in self.labels
```

Comment:
I am not going to lie, the fact that `true` / `false` is reported as a tag with `""` made this a very confusing read-through. We should consider moving those labels to tags themselves, either present or not, so the metadata.yaml is exactly what is being used here, and in BQ.

Reply:
I see. I opened https://github.com/mozilla/bigquery-etl/issues/878 for now and will address that later.